src/enclave-agent/src/main.rs

```diff
@@ -56,14 +59,47 @@ impl ImageService {
             validate::verify_id(&cid)?;
         }
 
+        let keyprovider_config = Path::new("/etc").join("ocicrypt_keyprovider_native.conf");
+        if !keyprovider_config.exists() {
+            let config = r#"
+            {
+                "key-providers": {
+                    "attestation-agent": {
+                        "native": "attestation-agent"
+                    }
```

Comment:
Why do we need to create keyprovider_config since kata-agent does not create this file?